src/Microsoft.ML.Data/Model/ModelOperationsCatalog.cs

```diff
@@ -182,12 +182,25 @@ public ITransformer Load(Stream stream, out DataViewSchema inputSchema)
         }
 
         /// <summary>
-        /// Load the model and its input schema from the stream.
+        /// Load the model and its input schema from the file.
+        /// </summary>
+        /// <param name="modelPath">Path to model.</param>
+        /// <param name="inputSchema">Will contain the input schema for the model. If the model was saved using older APIs
+        /// it may not contain an input schema, in this case <paramref name="inputSchema"/> will be null.</param>
```

Comment:
Is it worth calling out the behavior for older models having a null input schema? That should never be the case now right?